packages/webpack-config-single-spa/lib/webpack-config-single-spa.js

```diff
@@ -85,15 +85,16 @@ function webpackConfigSingleSpa(opts) {
     },
     devtool: "source-map",
     devServer: {
-      compress: true,
```

Comment:
I confirmed that this is the new default value in webpack-dev-server and that gzip compression still occurs with this change.